src/main/java/frc/robot/commands/dropping/DropperChooserCommand.java

```diff
@@ -32,27 +32,27 @@ private static SequentialCommandGroup getSequence(
         return switch (state) {
             case CONE_HIGH -> new DropSequence(
                 swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem,
-                ElevatorState.CONE_HIGH_DROP, .2, 0, 0, .2, 0, 0
+                ElevatorState.CONE_HIGH_DROP, .2, 0, 0, .2, 0
```

Comment:
why do we need the first .2sec wait here / do we even need the CONE_HIGH_DROP enum? what if we added a getDropPos() to the state enum instead, and by default it's just the same position?

Reply:
This would require completely changing the implementation of TiltedElevator so that instead of passing in ElevatorStates for setting extensions we would just pass in an extension value (basically rewriting all of TiltedElevator). The current code only needs the special drop positions for 2 things, and the other version would be worse imo. 

I set the .2 to 0 (I think it was .2 from when the tiltedElevatorCommand didn't end).

Reply:
i'm saying that each enum of ElevatorState could have a built-in property that is its drop offset, which can be 0 by default